Graphs/cpp/kruskal_mst.cpp

```diff
@@ -0,0 +1,101 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+/*
+Kruskal's algorithm is a minimum-spanning-tree algorithm which finds
+an edge of the least possible weight that connects any two trees
+in the forest.[1] It is a greedy algorithm in graph theory as it finds a
+minimum spanning tree for a connected weighted graph adding increasing cost
+arcs at each step.[1] This means it finds a subset of the edges that forms
+a tree that includes every vertex, where the total weight of all the edges
+in the tree is minimized. If the graph is not connected, then it finds a minimum
+spanning forest (a minimum spanning tree for each connected component).
+(c) Wikipedia.org
+
+This implementation is using DSU. O = M*log(N);
+M - edges, N - nodes.
+*/
+//FUNCTIONS

```

Comment:
No need for this comment 
We know that you are listing functions.
So you can remove //Functions.